examples/contract-transfer/lib.rs

```diff
@@ -183,18 +183,16 @@ pub mod give_me {
     #[cfg(test)]
     mod e2e_tests {
         use super::*;
-        use ink::env::e2e::*;
```

Comment:
This was previously required to bring the reexported `tokio` crate into scope for the `tokio::test` macro. In order to make this properly hygienic, I have replaced the usage of the `tokio::test` macro with explicit async execution (copied from `tokio::test`) https://github.com/paritytech/ink/pull/1429/commits/7e63d1ced263664f9a527a71d36c4d2e4bd9cd73